storage/drbd/pkg.yaml

```diff
@@ -21,6 +21,14 @@ steps:
         cp -r /rootfs/ /extensions-validator-rootfs/rootfs
         cp /pkg/manifest.yaml /extensions-validator-rootfs/manifest.yaml
         /extensions-validator validate --rootfs=/extensions-validator-rootfs --pkg-name="${PKG_NAME}"
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/drbd.spdx.json
+      version: {{ .DRBD_DRIVER_VERSION }}
+      cpes:
+        - cpe:2.3:a:linbit:drbd8:{{ .DRBD_DRIVER_VERSION }}:*:*:*:*:*:*:*
+        - cpe:2.3:a:linbit:drbd9:{{ .DRBD_DRIVER_VERSION }}:*:*:*:*:*:*:*
```

Comment:
`linbit:drbd8` exists, but looks like it's exclusively for `drbd` v8.x.y. We can assume that new vulnerability can be published under `linbit:drbd9`, but I am not sure if it should be put here.

CC @dsseng

Reply:
looks good, that's their fault of not following the CPE spec